projects/docs/docs/manifests/project.md

```diff
@@ -315,6 +315,12 @@ The following example shows the definition of an action that runs the `my_custom
 .pre(path: "my_custom_script.sh", name: "My Custom Script Phase", basedOnDependencyAnalysis: false)
 ```
 
+The following example shows the definition of an action that runs the `my_custom_script.sh` only for InstallBuilds:
```

Comment:
```suggestion
The following example shows the definition of an action that runs the `my_custom_script.sh` only for install builds:
```

(correct me if `InstallBuild`, as a name, is used, I might be wrong here)

Reply:
done

Reply:
done